int16 uav_id
time stamp
candidate[] candidates
